### ISSUE TITLE
 harvest.harvest.HarvestError: No JSON object could be decoded

### ISSUE THREAD

**Issue by mekhami** [lionheart/python-harvest#4]
I get this error: 

```
harvest.harvest.HarvestError: No JSON object could be decoded
```
on a successful delete request. I am not asking for a decodable JSON object, necessarily, but the code is attempting to return one.

My current 'solution' is to throw this in a try: except block and put the confirmation in the except block, but that's obviously suboptimal.

What's going on here, and what can I do to utilize this functionality correctly?

**Comment by gxela** [comment]
Is this still affecting you after version update?

**Comment by mekhami** [comment]
It should, as nothing relevant would have been changed.

**Comment by mekhami** [comment]
Ah, yes, here we are again.

Traceback (most recent call last):
  File "/usr/local/bin/sow", line 9, in <module>
    load_entry_point('Sow==0.1', 'console_scripts', 'sow')()
  File "build/bdist.macosx-10.10-x86_64/egg/sow/console.py", line 39, in main
  File "build/bdist.macosx-10.10-x86_64/egg/sow/console.py", line 33, in _main
  File "build/bdist.macosx-10.10-x86_64/egg/sow/commands.py", line 154, in delete

  File "/usr/local/lib/python2.7/site-packages/harvest/harvest.py", line 148, in delete
    return self._delete('/daily/delete/{0}'.format(entry_id))
  File "/usr/local/lib/python2.7/site-packages/harvest/harvest.py", line 160, in _delete
    return self._request('DELETE', path, data)
  File "/usr/local/lib/python2.7/site-packages/harvest/harvest.py", line 174, in _request
    raise HarvestError(e)
harvest.harvest.HarvestError: No JSON object could be decoded

**Comment by gxela** [comment]
this should be fixed in 1.0.4, do an update to 1.0.4

**Comment by mekhami** [comment]
Ah sorry, I had upgraded in a venv. It's now working the way I would expect it to. Thanks for your help!
